### PR TITLE
Decision mediator

### PR DESCRIPTION
**Pull request by NatalyMinziuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.BLL/Commands/Decision/DeleteDecisionAsyncCommand.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using EPlast.BLL.DTO;
+using MediatR;
+
+namespace EPlast.BLL.Commands.Decision
+{
+    public class DeleteDecisionAsyncCommand: IRequest
+    {
+        public int id { get; set; }
+        public DeleteDecisionAsyncCommand(int _id)
```
Comment (OleksandrBilan) [inline EPlast/EPlast.BLL/Commands/Decision/DeleteDecisionAsyncCommand.cs:9]: Please, don't use '_' symbol for input parameters, it is often used for private fields, but not parameters

File: EPlast/EPlast.BLL/Commands/Decision/DeleteDecisionAsyncCommand.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using EPlast.BLL.DTO;
+using MediatR;
+
+namespace EPlast.BLL.Commands.Decision
+{
+    public class DeleteDecisionAsyncCommand: IRequest
+    {
+        public int id { get; set; }
```
Comment (OleksandrBilan) [inline EPlast/EPlast.BLL/Commands/Decision/DeleteDecisionAsyncCommand.cs:8]: According to naming conventions all properties' names should start from uppercase letter

Comment (NatalyMinziuk) [reply EPlast/EPlast.BLL/Commands/Decision/DeleteDecisionAsyncCommand.cs]: Fixed

File: EPlast/EPlast.BLL/Commands/Decision/DeleteDecisionAsyncCommand.cs (hunk 1 above)
Comment (NatalyMinziuk) [reply EPlast/EPlast.BLL/Commands/Decision/DeleteDecisionAsyncCommand.cs]: Fixed

**Review by DimaDiD — APPROVED** [review]

**Review by bkondrat — APPROVED** [review]

**Review by VSBalanchuk — APPROVED** [review]

**Pull request merged**